tests/unittests/Foundation/NSPropertyListSerializationTests.mm

```diff
@@ -29,10 +29,7 @@
     });
     ASSERT_NE(nil, plistDict);
 
-    NSDateFormatter* formatter = [[NSDateFormatter new] autorelease];
-    ASSERT_NE(nil, formatter);
-    [formatter setDateFormat:@"yyyy-MM-dd'T'HH:mm:ss'Z'"];
-    NSDate* date = [formatter dateFromString:@"2014-12-15T19:48:38Z"];
+    NSDate* date = [NSDate dateWithTimeIntervalSinceReferenceDate:440365718]; // Mon Dec 15 2014 19:48:38 GMT
```

Comment:
Eh, it's a bit of a wash - I think I prefer not adding extraneous elements by bringing in Calendar here.

---
In reply to: [73243150](https://github.com/Microsoft/WinObjC/pull/792#discussion_r73243150) [](ancestors = 73243150)